src/index.ts

```diff
@@ -40,21 +53,66 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('operationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
```

Comment:
We haven't made the change suggested in my previous review that this no longer takes an array parameter.

Reply:
Whoops.